engine/entity/root/Window.cpp

```diff
@@ -185,9 +190,16 @@ void Window::render(glm::mat4 /*parentTransform*/) {
     glfwMakeContextCurrent(this->window);
     ImGui::SetCurrentContext(this->imguiContext);
 
+    // swap ImGui's config dir to prevent bundle crashes
+    std::string ConfigPath;
+    ConfigPath.append(Engine::getConfigDir());
+    ConfigPath.append("imgui.ini");
+    ImGui::GetIO().IniFilename = ConfigPath.c_str();
```

Comment:
This code is repeated seemingly unnecessarily

Reply:
whoops